src/Sylius/Behat/Page/Shop/Cart/CartSummaryPage.php

```diff
@@ -109,9 +109,109 @@ public function changeQuantity($productName, $quantity)
     /**
      * {@inheritdoc}
      */
+    public function isElementOnPage($elementName)
+    {
+        $items = $this->getElement('cart items')->findAll('css', 'thead > tr > th');
+        $summary = $this->getElement('cart summary')->findAll('css', 'tbody > tr > td > strong');
+        $elements = array_merge($items, $summary);
+
+        foreach($elements as $row) {
```

Comment:
Can you extract a common logic? This foreach is repeated at least three times in this class.